release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2867,4 +2865,28 @@ $ oc adm release info 4.11.7 --pullspecs
 [id="ocp-4-11-7-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-8"]
+=== RHBA-2022:6809 - {product-title} 4.11.8 bug fix update
+
+Issued: 2022-10-10
+
+{product-title} release 4.11.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2022:6809[RHBA-2022:6809] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:6808[RHBA-2022:6808] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.8 --pullspecs
+----
+
+[id="ocp-4-11-8-bug-fixes"]
+==== Bug fixes
+
+* Previously, in the monitoring stack, if you have enabled and deployed a dedicated Alertmanager instance for user-defined alerts, you cannot silence alerts in the Developer perspective in the OpenShift Container Platform web console. With this update, user-defined alerts can be silenced from the Developer perspective. (link:https://issues.redhat.com/browse/OCPBUGS-1790[*OCPBUGS-1790*])
```

Comment:
Same question as above in terms of "Developer" being capitalized.

Reply:
See comment above